modules/nf-core/scimap/mcmicro/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+
+nextflow_process {
+
+    name "Test Process SCIMAP_MCMICRO"
+    script "../main.nf"
+    process "SCIMAP_MCMICRO"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "scimap"
+    tag "scimap/mcmicro"
+
+    test("test-scimap-mcmicro") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				    [ id:'test' ], // meta map
+				    file(params.modules_testdata_base_path + 'imaging/downstream/cycif_tonsil_cell.csv', checkIfExists: true)
+				]
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.annotedDataCsv[0][1]).readLines()[0],  // unstable
+                    file(process.out.annotedDataH5ad[0][1]).name, // unstable
```

Comment:
Hol' up, `annotedDataCsv` could do with changing for two reasons:

1. It is not in snake_case
2. It should presumably be annotated rather than annoted

Reply:
Thank you!

I have simplified the output channel names.